releasenotes/notes/classical-store-e64ee1286219a862.yaml

```
@@ -1,5 +1,5 @@
 ---
-features:
+features_circuit:
```

In the template, this is listed at `features_circuits` (with an `s`). Which one is correct?

---

Oh, yeah good catch it has to match the config file.

```suggestion
features_circuits:
```